### DISCUSSION TITLE
Running file while hiding terminal?

### DISCUSSION THREAD

**Discussion by XKaos32X** [Tkd-Alex/Twitch-Channel-Points-Miner-v2#580] [Q&A]
So I love the program and have discord and analytics set up so I don't really need the console appearing anymore. Is there a way to hide the terminal besides setting the run file as a .pyw? I tried running it as .pyw file but no output was appearing in the discord or on the analytics localhost.

**Comment by mirronake** [comment]
On Windows
You can use VBS

Start.VBS
```
Set objShell = WScript.CreateObject("WScript.Shell")
objShell.Run("path to .bat"), 0, True
```

Start.bat
`python "run.py"`
